engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -317,9 +308,9 @@ private void generateSkyBand(FBO skyBand) {
         }
 
         skyBand.bind();
-        graphicState.setRenderBufferMask(skyBand, true, false, false);
+        setRenderBufferMask(skyBand, true, false, false);
 
-        setViewportTo(skyBand.dimensions());
+        setViewportToSizeOf(skyBand.dimensions());
```

Comment:
By the way, @emanuele3d important to note that I left some of the wrapped and unused methods for removal in the future, after all nodes are implemented. My motive is [making it work/clean/efficient ](http://c2.com/cgi/wiki?MakeItWorkMakeItRightMakeItFast)(in order).